dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/spirv/decorations/BuiltIn.java

```diff
@@ -15,13 +17,15 @@
 public class BuiltIn implements Decoration {
 
     private static final ExpressionFactory FACTORY = ExpressionFactory.getInstance();
-    private static final IntegerType TYPE_ARCH = TypeFactory.getInstance().getArchType();
 
     private final Map<String, List<String>> mapping = new HashMap<>();
 
     private int x = -1;
     private int y = -1;
     private int z = -1;
+    private int wgDimensionX = -1;
```

Comment:
Maybe rename it to something more generic, e.g. dimensionX? The BuiltIn class can also use these values later for other decorations.

Reply:
Renamed